### DISCUSSION TITLE
Make the code for video filters available?

### DISCUSSION THREAD

**Discussion by gustavostuff** [OnionUI/Onion#590] [Q&A]
Maybe I'm missing something but, is the code for [these filters](https://user-images.githubusercontent.com/16885275/173250366-dbaed067-640e-485d-8244-c62b9a7bd722.png) public? it'd be awesome to have the code available and use that in other platforms as well, after tweaking them (my plan is to use them with [LÖVE](https://love2d.org/) in a game I'm creating)

**Comment by Olywa** [comment]
Hi there, you can find these filters on your SD card inside `\RetroArch\.retroarch\filters\video\`, adding them to the same folders on another device/platform using Retroarch should enable to use them there. 

They are included in the Onion [source code here](https://github.com/OnionUI/RetroArch/tree/6351a511dfc812891a6b4e168b5d64c9386262e6/gfx/video_filters).
